src/test/java/org/jenkinsci/plugins/workflow/job/properties/PipelineTriggersJobPropertyTest.java

```diff
@@ -167,6 +170,30 @@ public void configRoundTrip() throws Exception {
         assertEquals("[null, false, null, true]", MockTrigger.startsAndStops.toString());
     }
 
+    @Test
+    public void triggerPresentDuringStart() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "triggerPresent");
+        r.configRoundtrip(p);
+        assertNull(getTriggerFromList(QueryingMockTrigger.class,
+                p.getTriggersJobProperty().getTriggers()));
+        JenkinsRule.WebClient wc = r.createWebClient();
+        String newConfig = org.apache.commons.io.IOUtils.toString(
+                PipelineTriggersJobPropertyTest.class.getResourceAsStream(
+                        "/org/jenkinsci/plugins/workflow/job/properties/PipelineTriggersJobPropertyTest/triggerPresentDuringStart.json"), "UTF-8");
+        WebRequest request = new WebRequest(new URL(p.getAbsoluteUrl() + "configSubmit"), HttpMethod.POST);
+        wc.addCrumb(request);
```

Comment:
I thought it possible to use HtmlForm for configuration, but not clear what json is really changing.